test/integration/managing_two_step_verification_test.rb

```diff
@@ -197,16 +198,15 @@ def exemption_message(initiator_name, reason)
         end
 
         context "when a exemption reason already exists" do
-          should "be able to see a link to edit exemption reason" do
-            user_requiring_2sv = create(:user, organisation: @organisation, reason_for_2sv_exemption: "user is exempt")
+          should "be able to see a to edit exemption reason and date" do
```

Comment:
`be able to edit exemption reason and date`

Reply:
👍  have fixed